### DISCUSSION TITLE
SOLVED: Another 404 issue on Pages

### DISCUSSION THREAD

**Discussion by joelindstrom** [community/community#22780] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve read through numerous posts on this, but can’t seem to find any with a similar issue.</p>
<p>See the attached screenshot. I’ve created a repository called joelindstrom, and my username is joelindstrom. So shouldn’t this page be published at <a href="https://joelindstrom.github.io/" rel="noopener nofollow ugc">https://joelindstrom.github.io/</a> in this case?</p>
<p>But as you can see on the screenshot, it’s at <a href="https://joelindstrom.github.io/joelindstrom/" rel="noopener nofollow ugc">https://joelindstrom.github.io/joelindstrom/</a> and if I don’t add the /joelindstrom/ directory I get the 404 message. Funnily, the links on my website believe they are the root directory so the don’t work either.</p>
<p>I’ve tried a few things, including deleting the repository and re-creating it. Same thing.</p>
<p>Any ideas? Am I missing something obvious?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/b/4bc61f707f07f0b129e9b42fd13d0c4f483d7f8b.png" data-download-href="/uploads/short-url/aOkhL6mxLWOuunUm0TXtIUUlQXp.png?dl=1" title="Screenshot 2020-10-07 at 14.56.47" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/70472124/181073118-2b37ff69-43d0-484e-8228-e74633005498.png" alt="Screenshot 2020-10-07 at 14.56.47" data-base62-sha1="aOkhL6mxLWOuunUm0TXtIUUlQXp" width="690" height="495" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/b/4bc61f707f07f0b129e9b42fd13d0c4f483d7f8b_2_690x495.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/b/4bc61f707f07f0b129e9b42fd13d0c4f483d7f8b_2_1035x742.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/b/4bc61f707f07f0b129e9b42fd13d0c4f483d7f8b_2_1380x990.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/b/4bc61f707f07f0b129e9b42fd13d0c4f483d7f8b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2020-10-07 at 14.56.47</span><span class="informations">1816×1304 239 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by sr229** [comment, marked as answer]
<p>Keep in mind you should name your repository <code>&lt;username&gt;.github.io</code> to assign the apex subdomain on the repository.</p>

**Comment by joelindstrom** [comment]
<p>Oh my, that <strong>solved</strong> it of course. I should’ve read the instructions more carefully.<br>
Thanks so much for the help!</p>
